tests/testthat/test.fd_inventory.R

```diff
@@ -0,0 +1,19 @@
+context("Loading internal package data")
+
+test_that("Inventory table", {
+  dat <- fd_inventory()
+
+  expect_is(dat$DBH_cm, "numeric")
+
+  # Health_status should be either L or D, or M
+  expect_true(all(dat$Health_status %in% c("L", "D", "M")))
+  # Canopy status has to be one of our four recognized values, or blank
+  expect_true(all(dat$Canopy_status %in% c("OD", "UN", "OS", "SA", "")))
```

Comment:
`NA` is much easier to deal with than `""`.
```suggestion
  expect_true(all(dat$Canopy_status %in% c("OD", "UN", "OS", "SA", NA_character_)))
```

Reply:
Hopefully we shouldn't have any blanks at all. Adding this to #10 for @atkinsjeff